lib/model.js

```diff
@@ -1429,13 +1429,14 @@ class Model {
   /**
    * Search for a single instance. This applies LIMIT 1, so the listener will always be called with a single instance.
    *
+   * __Alias__: _find_
+   *
    * @param  {Object}                    [options] A hash of options to describe the scope of the search
    * @param  {Transaction}               [options.transaction] Transaction to run query under
    * @param  {String}                    [options.searchPath=DEFAULT] An optional parameter to specify the schema search_path (Postgres only)
    *
-   * @see {Model#findAll}           for an explanation of options
-   * @return {Promise<Instance>}
-   * @alias find
+   * @see {@link Model#findAll}           for an explanation of options
+   * @return {Promise<Model>}
```

Comment:
JSDoc officially has the `@alias` tag: http://usejsdoc.org/tags-alias.html
I find it much nicer than defining it in the description. If it is not supported by doclets, maybe we can ask them to implement it?